src/plone/restapi/tests/test_content_get.py

```diff
@@ -23,8 +25,6 @@ class TestContentGet(unittest.TestCase):
     layer = PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
 
     def setUp(self):
-        if not HAS_AT:
```

Comment:
@arsenico13 does this test pass with AT?

Reply:
Here the test is using the dexterity layer. The ones with AT are at the bottom of the file, line 181.